[2024-04-13 18:35:01] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 2680fb919c17c10a [::1]:34070 failed checkpoint posthandshake: Connected to self
[2024-04-13 18:35:01] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 2680fb919c17c10a [::1]:20132 failed checkpoint posthandshake: Connected to self
[2024-04-13 18:35:04] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:09] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:14] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:19] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:24] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:29] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:31] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 2680fb919c17c10a [::1]:45140 failed checkpoint posthandshake: Connected to self
[2024-04-13 18:35:31] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 2680fb919c17c10a [::1]:20132 failed checkpoint posthandshake: Connected to self
[2024-04-13 18:35:34] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:39] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:44] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:49] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:54] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
[2024-04-13 18:35:59] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (2) nodes unable to establish P2P communication due to exceptions
